egs/aishell/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+# This script is modified based on swbd/s5c/local/nnet3/run_ivector_common.sh
+
```

Comment:
I think @danpovey was saying the minilibrispeech is a more recent script which should be preferable.